letsencrypt/README.rst

```diff
@@ -101,20 +100,71 @@ Further, if you force users to https, you'll need something like for nginx::
         rewrite ^   https://$server_name$request_uri? permanent;
     }
 
+A complete configuration file could look like this::
```

Comment:
a user who needs the below probably also needs instructions what to change where. You probably should add comments about that if you want this to be foolproof